CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ In addition, if you're interested in being credited, please add yourself to the
     <email>john.doe@citationstyles.org</email>
     <uri>https://citationstyles.org/</uri>
   </translator>
-  <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  <rights license="https://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
```

Comment:
This one won't work -- since we validate against the exact string here, we'd have to update all style files and there's really no reason to do that and thus make all last updated dates meaningless (especially since creativecommons.org automatically redirects to https anyway)

Reply:
OK - I'll just withdraw the PR.

Reply:
I'm actually happy to take the other two, especially since readthedocs doesn't enforce https; just revert this one.

Reply:
@koppor, thanks, I made the changes in the freeform text at https://github.com/citation-style-language/locales/commit/d0ee4d13c942648c2c88629aeb01f03a17fac72c.